lib/matplotlib/tests/test_check_figures_equal.py

```diff
@@ -0,0 +1,9 @@
+
+
+import matplotlib.pyplot as plt
+
+
+def test_check_figures_equal():
```

Comment:
You should use the check_figures_equal decorator. See

https://matplotlib.org/devdocs/api/testing_api.html?highlight=check_figures_equal#matplotlib.testing.decorators.check_figures_equal

Reply:
Hello, I have used the _check_figures_equal_ decorator in the test file.
Please review it.
Thanks.